test/test_kms_with_cli.py

```diff
@@ -62,6 +63,33 @@ def test_asymmetricKey_encrypt_decrypt(base_url, headers):
 
     print('====================test_asymmetricKey_encrypt_decrypt end===========================')
 
+def test_import_key(base_url, headers):
+    print('====================test_import_key start===========================')
+    key_GCM_256 = createkey.createkey(base_url, "EH_AES_GCM_256", "EH_EXTERNAL_KEY", "EH_KEYUSAGE_ENCRYPT_DECRYPT")
+    symmetricKey = [key_GCM_256]
+
+    aad = str(base64.b64encode("test".encode("utf-8")),'utf-8')
+
+    key = secrets.token_hex(16)
```

Comment:
used ?

Reply:
yes,the key is used to be encrypted by the rsa algorithm.